site/models/glossary-entry.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+require_once __DIR__ . '/default.php';
+
+class GlossaryEntryPage extends DefaultPage
+{
+	public function menuUrl(): string
+	{
+		return $this->link()->or($this->url())->value();
```

Comment:
```suggestion
		return $this->link()->toUrl() ?? $this->url();
```

For absolute URLs. And using `->toUrl()`.